client/components/jetpack/connection-health/index.tsx

```diff
@@ -45,6 +56,75 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		);
 	}
 
+	if ( errorType === FATAL_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					"Jetpack is unable to communicate with your site due to a critical error that has occurred there. Please check your site admin's email inbox for instructions."
+				) }
+				noticeActionHref={ localizeUrl(
+					'https://wordpress.com/support/why-is-my-site-down/#theres-a-critical-error-on-your-site'
+				) }
+				noticeActionText={ translate( 'Learn how to fix' ) }
+			/>
+		);
+	}
+
+	if ( [ USER_TOKEN_ERROR, BLOG_TOKEN_ERROR ].includes( errorType ) ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					'Jetpack is unable to communicate with your site due to a token error. Please reconnect Jetpack.'
+				) }
+				noticeActionHref={ localizeUrl(
+					'https://wordpress.com/support/why-is-my-site-down/#theres-an-issue-with-your-sites-jetpack-connection'
+				) }
+				noticeActionText={ translate( 'Learn how to fix' ) }
+			/>
+		);
+	}
+
+	if ( errorType === HTTP_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					'Jetpack is unable to communicate with your site due to a HTTP connection error. Please ensure that your site serves requests.'
```

Comment:
```suggestion
					'Jetpack is unable to communicate with your site because your site isn’t responding to requests.'
```

I think 'HTTP connection error' is too jargony. Also, 'Please ensure that your site serves requests' is better left for the support doc.

Reply:
Like in my previous comments, let's say `Jetpack can't communicate…`